src/command/click.rs

```diff
@@ -165,6 +165,22 @@ command!(
     }
 );
 
+command!(
+    Last,
+    "last",
+    "List target objects from the last executed query",
+    identity,
+    vec!["last"],
+    noop_complete!(),
+    no_named_complete!(),
+    |_, env, writer| {
+        if let Some(table) = env.get_last_table() {
```

Comment:
can we have an else here that just prints "no last objects to display" or similar?

Reply:
done